dlf/common/class.tx_dlf_helper.php

```diff
@@ -1123,17 +1113,7 @@ public static function saveToSession($value, $key) {
 	 */
 	public static function testInt($theInt) {
 
-		if (t3lib_utility_VersionNumber::convertVersionNumberToInteger(TYPO3_version) >= 6000000) {
-
-			// TYPO3 > 6.0
-			return t3lib_utility_Math::canBeInterpretedAsInteger($theInt);
-
-		} else {
-
-			// TYPO3 4.5 - 4.7
-			return \TYPO3\CMS\Core\Utility\GeneralUtility::testInt($theInt);
-
-		}
+		return t3lib_utility_Math::canBeInterpretedAsInteger($theInt);
```

Comment:
Maybe we should completely remove this helper function and replace all calls to it with a direct call to `t3lib_utility_Math::canBeInterpretedAsInteger`? The helper function was only introduced to have a downwards compatibility for TYPO3 4.5, which is no longer needed.